tests/e2e/tests/devfiles/JavaSpringBoot.spec.ts

```diff
@@ -33,30 +42,40 @@ const runTaskExpectedDialogue: string = 'Process 8080-tcp is now listening on po
 
 suite(`${stack} test`, async () => {
     suite(`Create ${stack} workspace`, async () => {
-        workspaceHandlingTests.createAndOpenWorkspace(stack);
+        // workspaceHandlingTests.createAndOpenWorkspace(stack);
+        test('Start Maven workspace using factory URL and vscode editor', async() => {
+            await createWorkspace.waitPage();
+            Logger.debug(`Fetching user kubernetes namespace, storing auth token by getting workspaces API URL.`);
+            await apiUrlResolver.getWorkspacesApiUrl();
+            workspaceHandlingTests.setWindowHandle(await browserTabsUtil.getCurrentWindowHandle());
+            // https://che-dogfooding.apps.che-dev.x6e0.p1.openshiftapps.com/dashboard/#/load-factory?url=https%3A%2F%2Fgithub.com%2Fche-samples%2Fjava-spring-petclinic%2Ftree%2Fdevfilev2&devWorkspace=https%3A%2F%2Fche-dogfooding.apps.che-dev.x6e0.p1.openshiftapps.com%2Fdevfile-registry%2Fdevfiles%2Fjava-web-spring%2Fdevworkspace-che-theia-next.yaml&storageType=persistent
+            // https://github.com/che-samples/java-spring-petclinic/tree/devfilev2&devWorkspace=https://che-dogfooding.apps.che-dev.x6e0.p1.openshiftapps.com/devfile-registry/devfiles/java-web-spring/devworkspace-che-theia-next.yaml&storageType=ephemeral&che-editor=che-incubator/che-code/insiders
+            await createWorkspace.startWorkspaceUsingFactory(`https://github.com/che-samples/java-spring-petclinic/tree/devfilev2&devWorkspace=${TestConstants.TS_SELENIUM_BASE_URL}/devfile-registry/devfiles/java-web-spring/devworkspace-che-theia-next.yaml&storageType=ephemeral&che-editor=che-incubator/che-code/insiders`);
```

Comment:
Let's extract `che-editor` name to parameter value as a constant.

It's interesting to know why `storageType=ephemeral`?